tools/run_tests/run_xds_tests.py

```diff
@@ -356,6 +413,39 @@ def wait_until_all_rpcs_go_to_given_backends(backends,
                                    num_rpcs,
                                    allow_failures=False)
 
+def wait_until_all_rpcs_fail(timeout_sec, num_rpcs):
```

Comment:
Is this new method really needed? Can we just call `wait_until_all_rpcs_go_to_given_backends_or_fail([])` instead?